test/podFailover/pod-failover-workload-chart/1pod1pvc.yaml

```diff
@@ -0,0 +1,24 @@
+storageClass:
+  name: azdisk-pod-failover-1pod1pvc-sc
```

Comment:
Lets add files driver pods as well. the dashboard also needs an update to include stateless and files driver data.